JITM: hide message 3 seconds after clicking the module act. CTA

jeherve opened the pull request:
#### Changes proposed in this Pull Request:

This will address part of #9462

I picked 3 seconds to give folks time to see that the button changed to an "Activated" status.

![screen recording 2018-12-12 at 05 49 pm](https://user-images.githubusercontent.com/426388/49885503-acc44380-fe37-11e8-9433-78d4047ea118.gif)


#### Testing instructions:

* Go to Jetpack > Settings and deactivate the Asset CDN.
* Refresh the page; you should see a JITM offering you to activate the feature.
* Click activate, and notice the message going through the different phases of the process (activating, activated), before to disappear.

#### Proposed changelog entry for your changes:
* Dashboard notices: automatically dismiss notices once a feature has been activated.

jeherve committed:
JITM: hide message 3 seconds after clicking the module act. CTA

This will address part of #9462

I picked 3 seconds to give folks time to see that the button changed to an "Activated"
status.

keoshi reviewed (dismissed):
Works great! Did you try with a `fadeOut`? It still causes the jump, but could be a neater way to make it go away. Just asking for curiosity, this looks good to me.

keoshi commented:
Not related to this PR, but I noticed that the site accelerator settings are not updated on-the-fly when I click this button. They still show up deactivated unless I refresh the page.

jeherve commented:
I believe they should update as well after a few seconds.

jeherve commented:
> Did you try with a fadeOut? 

I'll give it a try now.

jeherve committed:
Fade out JITM instead of just hiding it

@see https://github.com/Automattic/jetpack/pull/10960#pullrequestreview-184610594

jeherve commented:
@keoshi Here is how it looks after 814c0a4f7cf09187f30b25a2d4aab49693eb4437

![screen recording 2018-12-13 at 01 09 pm](https://user-images.githubusercontent.com/426388/49937921-94583580-fed8-11e8-8903-959c3d1628be.gif)

Is it better?

keoshi commented:
I kinda dig it like this, what do you think? Could even use less of a timeout since it now has the animation, but looks great!

jeherve commented:
I like it too. 

> Could even use less of a timeout since it now has the animation, but looks great!

Should it be 2 seconds instead of 3 then? I can make that change if needed.

keoshi commented:
@jeherve I believe the `slow` indicates 600 milliseconds, so 2 seconds sounds great to me!

oskosk reviewed (dismissed):
LGTM!

oskosk commented:
I approved this PR but not quite clear if it's good to go with 3 seconds or 2.

jeherve committed:
Fade out JITM after 2 seconds.

@see https://github.com/Automattic/jetpack/pull/10960#issuecomment-447023434

jeherve commented:
I changed it to 2 seconds in cf595242b2066b7fd3838077d322f378ff314dce.

@oskosk I'll let you merge if you like it :)
